spyder/app/mainwindow.py

```diff
@@ -1261,7 +1261,9 @@ def post_visible_setup(self):
 
         # Show history file if no console is visible
         if not self.ipyconsole.isvisible:
-            self.historylog.add_history(get_conf_path('history.py'))
+            self.historylog.add_history(
+                    "iPython history",

```

Comment:
Let's call this simply `History`.

Reply:
Also, its a moot point now but just for the record, its ``IPython``—they are actually pretty emphatic on that distinction.